libs/utils/utils.js

```diff
@@ -471,19 +473,39 @@ export function decorateSVG(a) {
   }
 }
 
+export function decorateModalImageLinks(el, a, btnSize) {
```

Comment:
None of this should be in utils. This is a huge edge case and every single visitor of a page should not have to pay for these bytes.

Reply:
@auniverseaway I have made the suggested changes and moved the JS outside into a different file and am now dynamically loading it when required.